tests/testthat/test-path.R

```diff
@@ -95,3 +95,16 @@ test_that("can write to a tar.gz file if the archive package is available", {
   expect_equal(res$path, "mtcars")
   expect_equal(res$size, 1281)
 })
+
+test_that("can read filepaths with non-ASCII characters", {
+  skip_on_cran()
+  ## chosen to be non-ASCII but
+  ## [1] representable in Windows-1252 and
+  ## [2] not any of the few differences between Windows-1252 and ISO-8859-1
+  ## a-grave + e-diaeresis  + Eth + '.csv'
+  tricky_filename <- "\u00C0\u00CB\u00D0\u002E\u0063\u0073\u0076"
+  file.copy(vroom_example("mtcars.csv"), tricky_filename)
```

Comment:
On principle, in terms of the unit test mindset, I suggest using the very smallest example file or even writing the file yourself. It can have 1 variable and 1 row. It just needs to exist. The only reason the readxl example uses `file.copy()` on a built-in example is because we can inline file creation `.xls` or `.xlsx`. But it's very easy to do create a small file that `vroom()` to read. You might find a good example of this pattern elsewhere in the vroom or readr tests (I haven't looked).